types/openlayers/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for OpenLayers v4.3.4
+// Type definitions for OpenLayers v4.6.0
```

Comment:
Can you re-enable lint rules in the appropriate `tslint.json` and fix up the version number here? We only accept major.minor (e.g. `4.3`) -  no patch, and no leading `v` (e.g. `v4.6.0`).

Reply:
Certainly.
Regarding lint rules though, I'm not sure which ones to enable. Removing everything except extending dtslint/dt.json leads to a waterfall of errors that I cannot handle at the moment. I see from other packages that some extra rules are not uncommon, is there any guidelines on which ones would prevent this PR to get merged? Especially considering that I didn't touch this file in the first place.

Reply:
I ran into a similar issue back in October when I submitted my PR. (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/21014) There are a lot of existing lint issues that should probably be addressed in a separate PR dedicated to that purpose.

Reply:
That's fair; just leave the lint rules in place, but fix up the version if that's okay.